### ISSUE TITLE
nautobot-server collectstatic Breakage

### ISSUE THREAD

**Issue by ichilton** [nautobot/nautobot#299]
### Environment
* Python version: 3.7.3
* Nautobot version: 1.0.0b3

### Steps to Reproduce
1. Follow the install instructions on: https://nautobot.readthedocs.io/en/latest/installation/nautobot/
2. When running `nautobot-server collectstatic`, it gives the following:

```
Configuration file does not exist at '/opt/nautobot/.nautobot/nautobot_config.py'
```

The documentation states `/opt/nautobot/nautobot_config.py` and `nautobot-server init` creates `/opt/nautobot/nautobot_config.py` - so it seems to be collectstatic looking in the wrong place.

3. Work around this by creating a symlink from `/opt/nautobot/.nautobot/nautobot_config.py` to `/opt/nautobot/nautobot_config.py`

4. Try `nautobot-server collectstatic` again and it gives:

```
Traceback (most recent call last):
  File "/opt/nautobot/lib/python3.7/site-packages/django/db/backends/base/base.py", line 219, in ensure_connection
    self.connect()
  File "/opt/nautobot/lib/python3.7/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/opt/nautobot/lib/python3.7/site-packages/django/db/backends/base/base.py", line 200, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/opt/nautobot/lib/python3.7/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/opt/nautobot/lib/python3.7/site-packages/django/db/backends/postgresql/base.py", line 187, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/opt/nautobot/lib/python3.7/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not connect to server: Connection refused
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 5432?
could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5432?
```

nautobot_config.py contains:

```
DATABASE = {
    'NAME': 'nautobot',
    'USER': 'nautobot',
    'PASSWORD': 'password_here',
    'HOST': '192.168.123.123',
    'PORT': '5432',
    'CONN_MAX_AGE': 300,
}
```

So it seems to be ignoring the configured host?

### Expected Behavior

`nautobot-server collectstatic` should refer to the configuration file in the correct place (`/opt/nautobot/nautobot_config.py`) and should use the configured database settings.

### Observed Behavior

`nautobot-server collectstatic` incorrectly looks for `/opt/nautobot/.nautobot/nautobot_config.py`  and then when given it, ignores the configured database.

EDIT: Running `nautobot-server runserver 0.0.0.0:8000 --insecure` is also trying to connect to postgresql on 127.0.0.1 instead of the configured host.

**Comment by jathanism** [comment]
Hello @ichilton. Thanks for reaching out. 

The location you're seeing at `/opt/nautobot/.nautobot` is the default location. Ff `NAUTOBOT_ROOT` isn't customized by the user, it defaults to `$HOME/.nautobot`. You may find the [Nautobot configuration guide](https://nautobot.readthedocs.io/en/latest/configuration/) helpful here.

It seems like you're missing a couple things:

- You do not appear to have set the [`NAUTOBOT_ROOT` environment variable](https://nautobot.readthedocs.io/en/latest/installation/nautobot/#update-the-nautobot-bashrc) (for example `export NAUTOBOT_ROOT=/opt/nautobot`). In the instructions we have you append this to your `.bashrc`.
- It looks like your database config came from NetBox. You'll need to make sure that you're using the correct settings for Nautobot which should be[ `DATABASES` (plural) and have an inner `default` key with your database information](https://nautobot.readthedocs.io/en/latest/configuration/required-settings/#databases), including `ENGINE` for the database driver. For example:

```
DATABASES = {
    'default': {
        'NAME': 'nautobot',                         # Database name
        'USER': 'nautobot',                         # PostgreSQL username
        'PASSWORD': 'awesome_password',             # PostgreSQL password
        'HOST': 'localhost',                        # Database server
        'PORT': '',                                 # Database port (leave blank for default)
        'CONN_MAX_AGE': 300,                        # Max database connection age
        'ENGINE': 'django.db.backends.postgresql',  # Database driver (Do not change this!)
    }
}
```

Give that a try!

**Comment by ichilton** [comment]
Thanks for the quick response!

The configuration thing was totally my fault - I did copy it from our Netbox config - I assumed it would be the same and at a glance it looked the same.

Then the other thing, I had set NAUTOBOT_ROOT in .bashrc.... but on reflection when I got that error, I was running it through SaltStack, which didn't have that ENV set - doh! - With hindsight, I fixed it with a symlink and then it wasn't until the other error that I tried running it manually.
